spec/yaml/swiftnav/sbp/ext_events.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Henry Hallam <henry@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.ext_events
+description: |
+  Messages reporting accurately-timestamped external events,
+  e.g. camera shutter time
+
+stable: False
+public: True
+include:
+  - types.yaml
+definitions:
+
+ - MSG_EXT_EVENT:
+    id: 0x0300
```

Comment:
I feel like the relationship between ext_events and time isn't really any stronger than e.g. the relationship between navigation and time.  But I don't really care or think it's terribly important.